tests/golden/dev/appcat/appcat/22_minio_prometheus_rule.yaml

```diff
@@ -0,0 +1,94 @@
+                    runbook_url: https://runbooks.prometheus-operator.dev/runbooks/kubernetes/kubepersistentvolumefillingup
+                    summary: PersistentVolume is filling up.
+                  expr: label_replace( bottomk(1, (kubelet_volume_stats_available_bytes{job="kubelet",
+                    metrics_path="/metrics"} / kubelet_volume_stats_capacity_bytes{job="kubelet",
+                    metrics_path="/metrics"}) < 0.15 and kubelet_volume_stats_used_bytes{job="kubelet",
+                    metrics_path="/metrics"} > 0 and predict_linear(kubelet_volume_stats_available_bytes{job="kubelet",
+                    metrics_path="/metrics"}[6h], 4 * 24 * 3600) < 0  unless on(namespace,
+                    persistentvolumeclaim) kube_persistentvolumeclaim_access_mode{
+                    access_mode="ReadOnlyMany"} == 1 unless on(namespace, persistentvolumeclaim)
+                    kube_persistentvolumeclaim_labels{label_excluded_from_alerts="true"}
+                    == 1) * on(namespace) group_left(label_appcat_vshn_io_claim_namespace)
+                    kube_namespace_labels, "name", "$1", "namespace", "vshn-minio-(.+)-.+")
+                  for: 1h
+                  labels:
+                    severity: warning
+            - name: minio-memory
+              rules:
+                - alert: minioMemoryCritical
+                  annotations:
+                    description: |-
+                      The memory claimed by the instance {{ $labels.name }} in namespace {{ $labels.label_appcat_vshn_io_claim_namespace }} has been over 85% for 2 hours.
+                        Please reducde the load of this instance, or increase the memory.
+                    runbook_url: https://hub.syn.tools/appcat/runbooks/vshn-generic.html#MemoryCritical
+                    summary: Memory usage critical
+                  expr: label_replace( topk(1, (max(container_memory_working_set_bytes{container="minio"})
+                    without (name, id)  / on(container,pod,namespace)  kube_pod_container_resource_limits{resource="memory"}
+                    * 100) > 85) * on(namespace) group_left(label_appcat_vshn_io_claim_namespace)
+                    kube_namespace_labels, "name", "$1", "namespace", "vshn-minio-(.+)-.+")
+                  for: 120m
+                  labels:
+                    severity: critical
+                    syn_team: schedar
+    providerConfigRef:
+      name: kubernetes
+metadata: {}
+name: prometheusrule
+patches:
+  - fromFieldPath: metadata.name
+    toFieldPath: metadata.name
+    transforms:
+      - string:
+          fmt: '%s-prometheusrule'
+          type: Format
+        type: string
+    type: FromCompositeFieldPath
+  - fromFieldPath: metadata.name
+    toFieldPath: spec.forProvider.manifest.metadata.namespace
+    transforms:
+      - string:
+          fmt: vshn-minio-%s
+          type: Format
+        type: string
+    type: FromCompositeFieldPath
```

Comment:
this is the file we're interested in, rest is just a moat after enabling minio for dev

Reply:
We need to make sure though that these alerts actually reach the our OpsGenie. `GeneratePrometheusNonSLORules` generates the alerts that are usually handled by the user workload monitoring. I guess we need to make sure that the organization on the minio instance namespaces are `vshn`.